src/main/java/uk/gov/ons/ctp/response/action/export/scheduled/ExportProcessor.java

```diff
@@ -86,12 +86,7 @@ public void processExport() {
             List<TemplateMapping> templateMappings = fileNameTemplateMappings.get(filename);
             for (TemplateMapping templateMapping : templateMappings) {
               if (templateMapping.getActionType().equals(ari.getActionType())) {
-                String filenamePrefix =
-                    filename
-                        + "_"
-                        + ari.getSurveyRef()
-                        + "_"
-                        + getExerciseRefWithoutSurveyRef(ari.getExerciseRef());
+                String filenamePrefix = filename;
```

Comment:
Could we just rename `filename` to `filenamePrefix` in the loop and get rid of this extra variable?